sdk/core/azure-core-tracing-opentelemetry/setup.py

```diff
@@ -63,6 +64,6 @@
     python_requires=">=3.8",
     install_requires=[
         "opentelemetry-api>=1.12.0",
-        "azure-core>=1.24.0",
+        "azure-core>=1.30.0"
```

Comment:
Black is probably complaining about the lack of a trailing comma here. Is there a need to bump the minimum core  version here?